pkg/cmd/api/api.go

```diff
@@ -77,7 +77,8 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			were added. Override the method with %[1]s--method%[1]s.
 
 			Pass one or more %[1]s--raw-field%[1]s values in "key=value" format to add
-			JSON-encoded string parameters to the POST body.
+			JSON-encoded string parameters to the POST body. Note that only simple strings, i.e.
```

Comment:
It just struck me that the phrase "JSON-encoded" might have been what was confusing for you, so I'd be down for changing that without adding any note about arrays for now. What do you think about making it clear that this flag is able to add only a single string value for each parameter? I think we can take out "JSON-encoded" to avoid confusion, and since JSON is an implementation detail anyway.

I'm thinking of something like:
> Pass one or more `--raw-field` values in "key=value" format to add string parameters to the request payload. To add non-string parameters, see `--field`. Note that adding request parameters will automatically switch the request method to POST. To send the parameters as a GET query string instead, use `--method GET`.

But you can tweak the wording if you think something is missing.

Reply:
Thx for your quick response! And in deed, the wording `JSON-encoded` threw me off since I was providing valid JSON-encoded strings for these "complex" objects :) Your suggestion is fine with me, let me change the PR.